facade/src/main/scala/reactST/reactTable/TableMaker.scala

```diff
@@ -128,42 +131,25 @@ case class TableMaker[D,
    */
```

Comment:
As the above comment notes, the order of plugins does seem to matter. Use of a `Set` for the plugins could be an issue for that. But, I think your use of an enum for the plugins could solve the problem and allow the user to be blissfully ignorant of the ordering requirement. An `order` could be added to the Plugin enum. The orders don't even have to be unique - ones for which it doesn't matter could have the same value.

The biggest issue of course, is knowing what the order requirements are. I'm not sure if there is any way to determine the order requirements other than trail and error. :(

Reply:
Oh my, I wasn't expecting this and I missed your comment.

I tried a few permutations that seemed to work, but it doesn't work for all of them then.

I'll follow your suggestion and add an `Order`, and start by assuming the order in which they appear in the documentation. If that doesn't work at any point, we can rearrange.

Reply:
That seems like a good starting point. At least we'll have a mechanism in place to fix it in when something breaks.